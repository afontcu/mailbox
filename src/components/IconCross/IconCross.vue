<template>
  <svg :viewPort="viewPort"
       :width="size"
       :height="size"
       :aria-label="ariaLabel"
       version="1.1"
       xmlns="http://www.w3.org/2000/svg"
  >
    <line x1="1"
          :y1="size - 1" 
          :x2="size - 1"
          y2="1" 
          stroke="currentColor" 
          stroke-width="2"
    />
    <line x1="1"
          y1="1" 
          :x2="size - 1"
          :y2="size - 1" 
          stroke="currentColor" 
          stroke-width="2"
    />
  </svg>
</template>

<script>
export default {
  name: 'IconCross',
  props: {
    size: { type: String, default: '12' },
    ariaLabel: { type: String, default: '' },
  },
  computed: {
    viewPort() {
      return `0 0 ${this.size} ${this.size}`
    },
  },
}
</script>
