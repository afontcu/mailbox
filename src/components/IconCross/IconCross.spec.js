import IconCross from './index.js'
import { mount } from '@vue/test-utils'

describe('IconCross.vue', () => {
  it('is a component', () => {
    const wrapper = mount(IconCross)
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  it('renders a snapshot', () => {
    const wrapper = mount(IconCross)
    expect(wrapper.html()).toMatchSnapshot('default')
  })
})
