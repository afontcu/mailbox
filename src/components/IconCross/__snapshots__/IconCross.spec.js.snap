// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconCross.vue renders a snapshot: default 1`] = `
<svg viewPort="0 0 12 12" width="12" height="12" aria-label="" version="1.1" xmlns="http://www.w3.org/2000/svg">
  <line x1="1" y1="11" x2="11" y2="1" stroke="currentColor" stroke-width="2"></line>
  <line x1="1" y1="1" x2="11" y2="11" stroke="currentColor" stroke-width="2"></line>
</svg>
`;
